.github/workflows/checks.yml

```diff
@@ -0,0 +1,55 @@
+
+name: PR Checks
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  checks:
+    name: Build, Test & Lint
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v4
+
+    - name: Set up Go
+      uses: actions/setup-go@v5
+      with:
+        go-version-file: go.mod
+
+    - name: Go Tidy
+      run: go mod tidy && git diff --exit-code
+
+    - name: Go Mod
+      run: go mod vendor
```

Comment:
Why is this here? 🤔

Reply:
I mean, if you don't vendor dependencies, then you don't need to call `go mod vendor`, all deps should be found in go cache. You need to run `go mod download` to get everything needed.

Reply:
I didn't vendor dependencies in https://github.com/six78/2-story-points-cli, and `golangci-lint` never complained 🤔